lib/rodauth/features/lockout.rb

```diff
@@ -23,10 +23,11 @@ module Rodauth
     notice_flash "Your account has been unlocked", 'unlock_account'
     notice_flash "An email has been sent to you with a link to unlock your account", 'unlock_account_request'
     redirect :unlock_account
+    response :unlock_account
     redirect(:unlock_account_request){default_post_email_redirect}
     redirect(:unlock_account_email_recently_sent){default_post_email_redirect}
     email :unlock_account, 'Unlock Account'
-      
```

Comment:
It would be best to avoid unnecessary whitespace changes when refactoring.

Reply:
removed